website/docs/release-notes.mdx

```diff
@@ -2,6 +2,274 @@
 title: Taqueria Releases
 ---
 
+## Taqueria v0.28.0
+| Details     |               |
+| ----------- | ------------- |
+|Release Date | Jan 26, 2023  |
+|Release Type | Significant         |
+|Release Page | [v0.28.0](https://github.com/ecadlabs/taqueria/releases/tag/v0.28.0) |
+
+### Summary of Impactful Changes
+
+#### Config Version Change
+
+Taqueria v0.28.0 includes a major change to the configuration schema.
+
+- The new schema version is *v2*, and is indicated in `.taq/config.json`:
+ ```json
+{
+    "version": "v2",
+    ...
+}
+```
+
+#### Environment Settings
+
+- Developer-specific environment settings are stored in local configuration files and should _not_ be stored in SCM. These files look like .taq/config.local.[envName].json.
+
+#### Zod Usage
+
+Our codebase has made use of [Zod](https://github.com/colinhacks/zod) for parsing and sanitizing for a while, but in this release, we now generate Zod schemas based on TypeScript type definitions.
+This makes the codebase much more accessible to third-party contributors that might not be as familiar with Zod.
+
+### New Features
+
+Here we highlight significant new features to Taqueria.
+
+#### Updated Workflow for SmartPy Developers
+
+We now support SmartPy as a first-class citizen in the Taqueria ecosystem.
+
+#### Test Stub Generation
+
+Taqueria can now create test stubs for Contracts. This capability has also been surfaced in the [VSCode Extension](https://marketplace.visualstudio.com/items?itemName=ecadlabs.taqueria-vscode).
+
+#### Taqueria Workflows
+
+Taqueria introduces the concept of "workflows", which facilitate project initialization:
+```
+taq init -w ligo`
+```
+
+This facility installs plugins typical to the workflow for a specific type of development. Taqueria currently ships with the following workflows:
+
+- ligo
+- smartpy
+- archetype
+- michelson
+
+#### Override Docker Images
+
+Taqueria now has the ability to override the particular docker images in use.
+
+Many plugins use docker images to render functionality. For instance, the v0.28.0 ligo plugin uses the `ligolang/ligo:0.57.0` image under the hood.
+
+Configure Taqueria alternatively by using the following environment variables:
+
+- TAQ_FLEXTESA_IMAGE
+- TAQ_LIGO_IMAGE
+- TAQ_ARCHETYPE_IMAGE
+
+For instance, to use LIGO v0.60 when compiling, do the following:
+```
```

Comment:
We need a line break between line 73 and 74. Fixing that got the build passing for me locally